react/src/components/lablupTalkativotUI/EndpointLLMChatCard.tsx

```diff
@@ -118,7 +120,7 @@ const EndpointLLMChatCard: React.FC<EndpointLLMChatCardProps> = ({
           popupMatchSelectWidth={false}
         />
       }
-      modelId={modelsResult?.data?.[0]?.id ?? 'custom'}
+      modelId={defaultModelId ?? modelsResult?.data?.[0]?.id ?? 'custom'}
```

Comment:
When `defaultModelId` exists, the user cannot change the model ID even if the endpoint has multiple models.